cmd/skaffold/app/cmd/flags.go

```diff
@@ -460,6 +460,22 @@ var flagRegistry = []Flag{
 		FlagAddMethod: "Var",
 		DefinedOn:     []string{"test", "deploy"},
 	},
+	{
+		Name:          "try-transparent-init",
+		Usage:         "If true, skaffold will try to create a config for the user's run if it doesn't find one",
+		Value:         &opts.TryTransparentInit,
+		DefValue:      true,
+		FlagAddMethod: "BoolVar",
+		DefinedOn:     []string{"debug", "dev", "run"},
+	},
+	{
+		Name:          "skip-confirmation",
```

Comment:
`--assume-yes`? also will this be globally applied to skip prompt from all user interactions?

Reply:
done